docs/networks/node-ops/nodes/node-operation/access-node-setup.md

```diff
@@ -257,15 +257,17 @@ If your node was selected as part of Step 3, you can now start your node.
 
 First you'll need to provision a machine or virtual machine to run your node software. Please see follow the [node-provisioning](./node-provisioning.md) guide for it.
 
-The access node can be run as a docker container with the following command by replacing `nodeid` with your node ID and adjusting the other parameters as per your setup.
+The access node can be run as a docker container with the following command by replacing `nodeid` with your node ID and adjusting the other parameters as per your setup. 
+
+Be sure to use the version tag corresponding to the current network version (see [here](https://github.com/onflow/flow-go/releases) for version releases).
 
 ```shell
 docker run --rm \
   -v $PWD/bootstrap:/bootstrap:ro  \
   -v $PWD/data:/data:rw \
   --name flow-go \
   --network host \
-  gcr.io/flow-container-registry/access:v0.29.8 \
+  gcr.io/flow-container-registry/access:$VERSION \ # Use the version for the current spork, from https://github.com/onflow/flow-go/releases
```

Comment:
Need a more clear note to inform the user what $VERSION means.  The user will see this screen shot.
Are you saying use the `tag` value or use "Mainnet 24" string, I think simply adding "ex: v0.29.8" is enough.
![image](https://github.com/onflow/docs/assets/3970376/27bfed54-28af-4e3f-830a-788e0c534cc4)

Reply:
Changed the text above the command to be more explicit:

> Be sure to set `$VERSION` below to the version tag (eg. `v1.2.3`) corresponding to the current network version (see [here](https://github.com/onflow/flow-go/releases) for version releases). Set `$NODEID` to your node's ID (see [Generate Your Node Identity](#generate-your-node-identity) section above).

Also updated the example command below to include an sample version tag.